src/gsi/read_dbz_nc.f90

```diff
@@ -337,6 +338,7 @@ subroutine read_dbz_nc(nread,ndata,nodata,infile,lunout,obstype,sis,hgtl_full,no
 
   call w3fs21(iadate,mins_an)  !mins_an -integer number of mins snce 01/01/1978
   rmins_an=mins_an             !convert to real number
+  timeb=real(mins_an-iwinbgn,r_kind)
```

Comment:
Ok, I think now, we agree timeb should be the observation time relative to the begin of the time window.  
You could change that wrong comment .  And, could you confirm in "timeb=real(mins_an-iwinbgn,r_kind)" , iwindbgn is in the same time coordinates as mins_an?

Reply:
Good. Thanks.